src/App/Fossa/API/BuildWait.hs

```diff
@@ -24,7 +24,7 @@ import Control.Effect.StickyLogger (StickyLogger, logSticky')
 import Control.Timeout (Cancel, checkForCancel)
 import Data.Text (Text)
 import Effect.Logger (Logger, pretty, viaShow)
-import Fossa.API.Types (ApiOpts, Issues (..))
+import Fossa.API.Types
```

Comment:
Don't use implicit imports:
https://github.com/fossas/fossa-cli/blob/master/docs/contributing/STYLE-GUIDE.md#dont-use-implicit-blanket-imports

We should also clean up the pre-existing implicit imports here, if you're using VSCode there's a code action for making imports explicit automatically (you'll have to run the formatter afterward).